gee_tools/datasources/interface.py

```diff
@@ -20,11 +22,13 @@ def build_img_coll(self, **kwargs):
         raise NotImplementedError
 
     @abc.abstractmethod
-    def get_img_coll(self):
+    def get_img_coll(self, **kwargs):
```

Comment:
The way I was passing these kind of arguments previously with through the constructor.  If you think it makes more sense to pass arguments through the get_img_coll function, then lets remove the **kwargs from the constructor.  We should agree on what the constructor arguments are supposed to be.